src/routes/safe/store/selectors/index.ts

```diff
@@ -79,14 +79,14 @@ export const safeCancellationTransactionsSelector = createSelector(
   safeParamAddressFromStateSelector,
   (cancellationTransactions, address) => {
     if (!cancellationTransactions) {
-      return List([])
+      return Map()
     }
 
     if (!address) {
-      return List([])
+      return Map()
     }
 
-    return cancellationTransactions.get(address) || List([])
+    return cancellationTransactions.get(address) || Map({})
```

Comment:
https://immutable-js.github.io/immutable-js/docs/#/Map/get

`.get` has param for `notSetValue`, you could use instead of `|| Map({})`

Reply:
Nice!